helix-term/tests/test/movement.rs

```diff
@@ -424,7 +450,7 @@ async fn select_mode_tree_sitter_next_function_is_union_of_objects() -> anyhow::
             "]fv]f",
             helpers::platform_line(indoc! {"\
                 /// Increments
-                #[fn inc(x: usize) -> usize { x + 1 }
+                fn inc(x: usize) -> usize { x + 1 }#[
```

Comment:
This behavior change makes extending worse in this scenario. Instead of this patch you may just want to remap the `]f` and related bindings so that they call `flip_selections`